django/apps/registry.py

```diff
@@ -45,8 +46,10 @@ def __init__(self, installed_apps=()):
         # Lock for thread-safe population.
         self._lock = threading.Lock()
 
-        # Pending lookups for lazy relations.
-        self._pending_lookups = {}
+        # Maps ("app_label", "modelname") tuples to lists of functions to be
+        # called when the corresponding model is ready. See this class'
+        # lazy_model_operation and do_pending_operations for details.
```

Comment:
"methods for details"
(I also often write the names as `lazy_model_operation()` (with parenthesis) so it's obvious you're referring to a method and not an attribute.